### PR TITLE
fix(help command): Display the 'default' script when using the 'help' command

### PR DESCRIPTION
**Pull request by wabirached**
Display the 'default' script when using the 'help' command alongside all the other scripts.

Issue #130

**What**: Display the 'default' script when using the 'help' command alongside all the other scripts.

**Why**: Hotfix for issue #130 

**PS: when using `npm start commit`, the linter is modifying lines that were untouched prior to the commit.**

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -275,6 +275,13 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "wabirached",
+      "name": "Wissam Abirached",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/3534924?v=3",
+      "profile": "https://designingforscale.com",
+      "contributions": []
```
Comment (kentcdodds) [inline .all-contributorsrc]: Do you want to add `code` and `test` to this? Then run `npm start generateContributors` to regenerate the table :+1:

**Comment by kentcdodds** [comment]
> PS: when using npm start commit, the linter is modifying lines that were untouched prior to the commit.

Yeah, sorry about that :)

**Comment by wabirached** [comment]
My pleasure, Kent! Glad to help.

**Comment by kentcdodds** [comment]
Thank you! This is great!

**Pull request merged**